ios/simlocation/simlocation.go

```diff
@@ -162,7 +162,10 @@ func SetLocationGPX(device ios.DeviceEntry, filePath string) error {
 				pointLat := point.PointLatitude
 
 				// Set the current point location by its latitude and longitude
-				SetLocation(device, pointLat, pointLon)
+				err = SetLocation(device, pointLat, pointLon)
```

Comment:
nice! I am also constantly looking for those. Go makes it too easy to miss those imo. 
And it can make the library/CLI tool really difficult to use.

Reply:
Yeah, I know... sometimes I just skip handling the error while testing out code and then completely forget about it.. 😆

Reply:
haha same for me 😅